test/tests/extras.test.ts

```diff
@@ -34,14 +34,20 @@ describe(`Extra Tests`, () => {
     });
 
     describe(`ts-node register script`, () => {
+      /** Yarn sometimes outputs bold text, which makes these tests flakey */
```

Comment:
These tests were failing for me when I ran them locally. The output of `yarn` seems to include control characters to bold the string `null`, which is why I'm stripping the string with this method.

```
 FAIL  tests/extras.test.ts (8.413 s)
  ● Extra Tests › Built Tests › ts-node register script › Works with --transpileOnly

    expect(received).toMatch(expected)

    Expected pattern: /^null($|\r?\n)/m
    Received string:  "null
    "

      37 |       test(`Works with --transpileOnly`, () => {
      38 |         const res = execSync("yarn g:ts-node --transpileOnly src/index.ts", { cwd: projectRoot }).toString();
    > 39 |         expect(res).toMatch(/^null($|\r?\n)/m);
         |                     ^
      40 |       });
      41 |
      42 |       test(`Works with --typeCheck`, () => {

      at Object.<anonymous> (tests/extras.test.ts:39:21)

  ● Extra Tests › Built Tests › ts-node register script › Works with --typeCheck

    expect(received).toMatch(expected)

    Expected pattern: /^null($|\r?\n)/
    Received string:  "null
    "

      42 |       test(`Works with --typeCheck`, () => {
      43 |         const res = execSync("yarn g:ts-node --typeCheck src/index.ts", { cwd: projectRoot }).toString();
    > 44 |         expect(res).toMatch(/^null($|\r?\n)/);
         |                     ^
      45 |       });
      46 |     });
      47 |   });

      at Object.<anonymous> (tests/extras.test.ts:44:21)
```